### PR TITLE
task(docs): add more information to docs

### PR DESCRIPTION
**Pull request by cerebrl**
## Summary

Just adding more content to our docs within the Contributing section. None of this content is final as I'm just trying to get words onto the the page as I think of them.

## Bug fix

There was a very small visual bug with the animated radio buttons not being centered, so there's a small fix for that.

### CODE REVIEW

File: themes/default/compositions.cjs [hunk 1]
```diff
@@ -274,9 +274,11 @@ module.exports = (config, theme) => ({
   '.animated-radio': {
     position: 'relative',
     cursor: 'pointer',
-    fontSize: '1.5em',
-    height: '1em',
-    width: '1em',
+    // fontSize: '1.5em',
+    // replacing `1em` with `24px`
+    // using `em` results in misaligned circles due to half-pixel (e.g. 22.5px) grid alignment issue
```
Comment (cerebrl) [inline themes/default/compositions.cjs:279]: When using `em`, the browser may calculate the width and height to be a non-integer value (e.g. 22.5 pixels). That can lead to misalignment as these items fall onto the pixel grid. Using a round, even number instead, like `24px` makes it easier to create a predictable alignment.

**Review by daveadams56 — APPROVED** [review]
Overall looks good - just a couple of very minor points if you get time.

File: themes/default/compositions.cjs (hunk 1 above)
Comment (cerebrl) [reply themes/default/compositions.cjs:279]: Good question. `fontSize` was the attribute driving the base value that informed the `em` unit value in `height` and `width`, which was the underlying issue. Switching to absolute values removed the need for declaring a `fontSize`.

File: src/routes/docs/widget/roadmap/+page.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-## Future APIs (not implemented)
+# Future APIs (not implemented)
```
[the remark this answers is not in the document]
Comment (cerebrl) [reply src/routes/docs/widget/roadmap/+page.md:1]: Good point. I'll make a note of this locally and include it in a different PR.

**Pull request merged**